src/util/save-app-properties.ts

```diff
@@ -0,0 +1,18 @@
+import { getJiraClient } from "~/src/jira/client/jira-client";
+
+export const getAppKey = (): string => {
```

Comment:
/nit this looks like a handy function to have in a generic utils or its own file 🤷

Reply:
I only did it this way coz in the uninstall event it's calling the delete method which is diff to what's being called in the main function here. Added this guy to try and at least reduce some of the repeated code there but happy to kill this and just add those lines over there if you think that's better.

Reply:
same things, leave it there, if we need it we can refactor then, you have already broken it outside of the core function so its easy enough for next person to move it if they need it.